doc/telescope.txt

```diff
@@ -1459,7 +1459,7 @@ builtin.lsp_dynamic_workspace_symbols({opts}) *builtin.lsp_dynamic_workspace_sym
 
 
 builtin.diagnostics({opts})                            *builtin.diagnostics()*
-    Lists diagnostics for current or all open buffers
```

Comment:
Documentation is auto-generated and therefore any changes should stem from the corresponding comments. In this case from `/telescope/builtin/init.lua` at `builtin.diagnostics`.